ts/components/core/variables/IOColors.ts

```diff
@@ -72,7 +85,6 @@ export const IOColors = asIOColors({
   milderGray: "#5F6F80",
   bluegrey: "#475A6D",
   bluegreyDark: "#17324D",
```

Comment:
Just a curiosity, why these colors are not following the naming convention used for the other colors? Like "bluegrey-500" and "bluegrey-700", for example.

Reply:
Because they are legacy. It doesn't make sense to change them now if we're going to use the new color palette in the short to medium term.